src/content/en/fundamentals/accessibility/01-focus/focus-five.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: shared/narrow
+title: "Using tabindex"
+description: "Modifying the DOM order with tabindex"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 5
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+The default tab order provided by the DOM position of native elements is convenient, but there are times when you'll want to modify the tab order, and physically moving elements in the HTML isn't always an optimal, or even a feasible, solution. For these cases you can use the `tabindex` HTML attribute to explicitly set an element's tab position.
+
+`tabindex` can be applied to any element &mdash; although it is not necessarily useful on every element &mdash; and takes a range of positive integer values. Using `tabindex`, you can specify an explicit order for focusable page elements, insert an otherwise unfocusable element (such as a button made from a `div`) into the tab order, and remove elements from the tab order. Like `focus`, `taborder` can also be controlled programmatically. See <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex" target="_blank">this MDN page</a> for more details. 
```

Comment:
Suggest adding a list after this bit: "...and remove elements from the tab order."

- `tabindex="0"`: Inserts element in the natural tab order and element can be focused by calling its `focus()` method
- `tabindex="-1"`: Removes an element from the natural tab order. But element can still be focused by calling its `focus()` method
- `tabindex="5"`:  Any tabindex greater than 0 will jump the element to the front of the natural tab order. If there are multiple elements with a tabindex greather than 0, the tab order will start from the lowest value that is greater than zero, and work its way up. Using tabindex greater than 0 is considered an **anti-pattern**.